style.css

```diff
@@ -544,3 +556,184 @@ html {
   flex-grow: 0;
   margin: 24px 0;
 }
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 24px;
+  width: 375px;
+  height: 844px;
+  margin-top: 58px;
+}
+
+.form-heading {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px 0;
+  width: 327px;
+  height: 216px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  width: 327px;
+  height: 330px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.form-inputs {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  width: 327px;
+  padding: 0;
+}
+
+.submit-button {
+  padding: 12px;
+  width: 131px;
+  height: 48px;
+  background: #ff6b00;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.submit-button:hover {
+  background: #ff5100;
+  padding: 14px;
+  width: 135px;
+}
```

Comment:
- [ ] Please add the right hover styling and kindly add `active` pseudo-class (when it's pressed) styling as well.

![image](https://user-images.githubusercontent.com/68796232/159749239-de3e7764-bda7-4202-92bc-786d6cc80cee.png)

Reply:
I added the active pseudo-class but I don't know how to get hover values from Figma so I used approximate values